### PR TITLE
email verification

### PR DESCRIPTION
**Pull request by lonewolf9277**
- isVerified field added to user model
- Route added to set isVerified to true

### CODE REVIEW

File: src/app/api/user/verifyEmail/route.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+// Connect the database
+dbConnect();
+
+// Defining zod schema (no export here)
+const ZodValidation = z.object({
+  email: z.string().email(),
+});
+
+// Post the data
+export async function POST(request: NextRequest) {
```
Comment (Shu12388y) [inline src/app/api/user/verifyEmail/route.ts:16]: @freelancer79 We have to get the email for params object. Not by a request body

**Review by Shu12388y — CHANGES_REQUESTED** [review]
@freelancer79 resolve the issue

File: src/app/api/user/verifyEmail/route.ts (hunk 1 above)
Comment (lonewolf9277) [reply src/app/api/user/verifyEmail/route.ts:16]: Could you explain a bit more. I thought you asked me to add a route.

Comment (Shu12388y) [reply src/app/api/user/verifyEmail/route.ts:16]: We do not get the user email as a params object for the route file but request body. Check out nextjs docs how to can add dynamic routes

Comment (lonewolf9277) [reply src/app/api/user/verifyEmail/route.ts:16]: Alright got it. I'll complete this today.

**Comment by lonewolf9277** [comment]
Done please check.

**Pull request merged**